test/unit/org/apache/cassandra/service/accord/api/AccordKeyTest.java

```diff
@@ -59,26 +59,26 @@ public static IPartitioner partitioner(TableId tableId)
     public void partitionKeyTest()
     {
         DecoratedKey dk = partitioner(TABLE1).decorateKey(ByteBufferUtil.bytes(5));
-        PartitionKey pk = new PartitionKey(TABLE1, dk);
+        PartitionKey pk = new PartitionKey("ks", TABLE1, dk);
```

Comment:
Completely insignificant, but is there a reason to use `"ks"` here (that exists) vs `""` elsewhere below?

Reply:
Yes, TABLE1 has a keyspace of "ks" and we serialise only the tableId and grab the keyspace from its metadata, so we deserialise ks.